exercises/helping/problem.md

```diff
@@ -1,41 +1,46 @@
-Create a server which responds to requests to `/?name=Helping&suffix=!` using
```

Comment:
It looks like you have touch the wrong file. Shouldn't be the `problem.fr.md` file?